src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -413,6 +419,9 @@ public static String jobConfig(String jobKey, String cfgName) {
             this.put(CACHE_ANON_CERT_CONTENT_TTL, "120000"); // milliseconds
             this.put(CACHE_ANON_CERT_CONTENT_MAX_ENTRIES, "10000");
 
+            this.put(PAGING_SIZE, "10");
+            this.put(MAX_PAGING_SIZE, "10000");
```

Comment:
I think we should reduce this. The `PageRequest.DEFAULT_PER_PAGE` was 10. And out of the 3 endpoints that specified their own max values, two of them had the max at 1000, and only one at 10000.
1000 seems like a reasonable max page value